src/ontology/cl-edit.owl

```diff
@@ -4571,15 +4571,15 @@ EquivalentClasses(obo:CL_0000104 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeV
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FMA:67305") Annotation(oboInOwl:hasDbXref "GOC:tfm") obo:IAO_0000115 obo:CL_0000105 "Neuron with two neurites that are fused grossly when they protrude from the soma and bifurcate a short distance from the soma.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000105 "FMA:67305")
 AnnotationAssertion(rdfs:label obo:CL_0000105 "pseudounipolar neuron")
-SubClassOf(obo:CL_0000105 obo:CL_0000540)
+SubClassOf(obo:CL_0000105 obo:CL_2000032)
 
 # Class: obo:CL_0000106 (unipolar neuron)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "FMA:67278") Annotation(oboInOwl:hasDbXref "GOC:nv") obo:IAO_0000115 obo:CL_0000106 "Neuron with one neurite that extends from the cell body.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000106 "FMA:67278")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_0000106 cl:BDS_subset)
 AnnotationAssertion(rdfs:label obo:CL_0000106 "unipolar neuron")
-SubClassOf(obo:CL_0000106 obo:CL_0000540)
+SubClassOf(obo:CL_0000106 obo:CL_2000032)
```

Comment:
"A unipolar neuron is a neuron in which only one process, called a [neurite](https://en.wikipedia.org/wiki/Neurite), extends from the [cell body](https://en.wikipedia.org/wiki/Soma_(biology)). The neurite then branches to form dendritic and axonal processes. Most neurons in the central nervous systems of [invertebrates](https://en.wikipedia.org/wiki/Invertebrate), including insects, are unipolar.[[2]](https://en.wikipedia.org/wiki/Unipolar_neuron#cite_note-2) The cell bodies of invertebrate unipolar neurons are often located around the edges of the [neuropil](https://en.wikipedia.org/wiki/Neuropil), in the so-called cell-body rind.[[3]](https://en.wikipedia.org/wiki/Unipolar_neuron#cite_note-3)" 

I found these information showing that they are also in CNS . https://en.wikipedia.org/wiki/Unipolar_neuron

Reply:
we are not covering invertebrates in this classification, they will be classified differently